src/main/java/com/launchdarkly/client/LDUser.java

```diff
@@ -107,11 +116,14 @@ JsonPrimitive getAnonymous() {
   }
 
   JsonElement getCustom(String key) {
-    return custom.get(key);
+    if (custom != null) {
```

Comment:
By construction, custom is not null. Perhaps annotating it as `@NonNull` makes more sense?